Sources/ArcGISToolkit/Components/Augmented Reality/Utilities/WorldTrackingSceneView.swift

```diff
@@ -192,27 +167,24 @@ public struct WorldScaleGeoTrackingSceneView: View {
         }
     }
     
-    /// If necessary, updates the scene view's camera controller for a new location coming
-    /// from the location datasource.
+    /// Updates the scene view's camera controller with a new location coming from the
+    /// location data source and resets the AR session when using world-tracking configuration.
     /// - Parameter location: The location data source location.
-    @MainActor
-    private func updateSceneView(for location: Location) {
+    private func updateWorldTrackingSceneView(for location: Location) {
         // Do not update the scene view when the coaching overlay is in place.
         guard !coachingOverlayIsActive else { return }
         
-        // Do not use cached location more than 10 seconds old.
-        guard abs(lastLocationTimestamp?.timeIntervalSinceNow ?? 0) < 10 else { return }
+        // Update if the location is more than 10 seconds old.
```

Comment:
Maybe I can revert to the old comment.

I mean "refresh the stale location if it is older than 10 seconds". It may be clearer than the current comment.